Spanner/tests/Unit/Session/lock_test_process.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Google\Cloud\Spanner\Tests\Unit\Session;
+
+use DG\BypassFinals;
+use Google\Cloud\Spanner\Session\SessionCache;
+use Google\Cloud\Spanner\V1\Client\SpannerClient;
+use Prophecy\Argument;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+
+/**
+ * Runs a process which is designed to wait while a session is acquired.
+ */
+if (count($argv) !== 2) {
+    die('Usage: lock_test_process.php DATABASE_NAME' . PHP_EOL);
+}
+
+if (file_exists(__DIR__ . '/../../../vendor/autoload.php')) {
+    // google/cloud-spanner autoload
+    require __DIR__ . '/../../../vendor/autoload.php';
+} elseif (file_exists(__DIR__ . '/../../../../vendor/autoload.php')) {
+    // google/cloud autoload
+    require __DIR__ . '/../../../../vendor/autoload.php';
+}
+
+BypassFinals::enable();
```

Comment:
Was comment to move this to a bootstrap file, but is this here because of this being related to a locking mechanism? Or we can move it to a bootstrap file?